app/views/invitations/_user_invitations.html.erb

```diff
@@ -1,32 +1,35 @@
 <% unless invitations.blank? %>
-  <table summary="<%= ts('List of your invitation tokens and information regarding who you shared them with, along with the option to share unused tokens.') %>">
-    <caption><%= ts('Invitation Information') %></caption>
-	  <thead>
-	    <tr>
-        <th scope="col"><%= ts('Token') %></th>
-        <th scope="col"><%= ts('Sent To') %></th>
-        <th scope="col"><%= ts('User Name') %></th>
-        <th scope="col"><%= ts('External Author') %></th>
-        <th scope="col"><%= ts('Copy Link') %></th>
+  <table summary="<%= t(".list_of_your_invitations") %>">
+    <caption><%= t(".invitation_information") %></caption>
```

Comment:
I'd love it if we could use the table key naming scheme established in various admin places, e.g. https://github.com/otwcode/otwarchive/blob/74655f9d9365359963ec368f184cbc74f6299427/app/views/admin/api/index.html.erb#L21-L49